qa/rgw/store/sfs/build-radosgw.sh

```diff
@@ -29,11 +29,14 @@ WITH_UBSAN=${WITH_UBSAN:-"OFF"}
 
 NPROC=${NPROC:-$(nproc --ignore=2)}
 
+CC=${CC:-"gcc-11"}
```

Comment:
Suggest not pinning the version number. On build container update it might no longer exists.  Running on tumbleweed gcc-13 would be available.

Reply:
I've pinned it to the version that we had previously, because that's the version pinned in the build container, which is AFAIK the minimum version required to build this source tree. If it changes there, the build container can set the environment variables to match the version it installs.
Frankly this is the exact reason why I think it's hair brained to maintain build infrastructure in a separate repository. Although what we're doing right now, maintaining _part_ of the build and test infra there and _part_ of it here is even worse.